build.gradle

```diff
@@ -139,7 +141,7 @@ configurations {
 }
 
 dependencies {
-    def modelixCoreVersion = "2.1.9"
+    def modelixCoreVersion = "1.10.3"
```

Comment:
Why is the modelix version downgraded?

Reply:
Probably because the migration was started with an older state of the branch.